TelePharo-Core.package/TlpRemotePharo.class/class/resetCalypsoAnnouncers.st

```diff
@@ -0,0 +1,9 @@
+instance creation
+resetCalypsoAnnouncers
+	"This method resets all announcers of Calypso. When the network is broken, the server keeps the old announcers, so we need reset them"
+	
+	| objectsWithAnnoucers announcersWithSubscribers |
+	
+	objectsWithAnnoucers := ClyQueryResult allSubInstances select: [ :each | each announcer isNotNil ].
+	announcersWithSubscribers := objectsWithAnnoucers 	collect: [ :each | each announcer ].
+	announcersWithSubscribers do: [ :each | each subscriptions reset ]
```

Comment:
It should be simple:
```Smalltalk
ClyNavigationEnvironment currentImage queryCache removeAll.
```
No need to check subscriptions. Just clean the cache

Reply:
And following my first comment it can be method on Calypso side:
```Smalltalk
ClyFullBrowser class>>cleanupEnvironmentFromRemoteTool: aRemoteBrowser
   ClyNavigationEnvironment currentImage queryCache removeAll.
```

Reply:
I created these 2 methods in server-side, but they didn't clean up and the error start again when I reproduce this scenario https://github.com/pharo-iot/PharoThings/issues/42